src/librbd/cache/pwl/AbstractWriteLog.cc

```diff
@@ -1443,8 +1446,8 @@ bool AbstractWriteLog<I>::check_allocation(C_BlockIORequestT *req,
                                  << ", have " << m_free_log_entries << ") "
                                  << *req << dendl;
       alloc_succeeds = false;
-      no_space = true; /* Entries must be retired */
-    }
+      no_space = true; // Entries must be retired
+    }*/
```

Comment:
This is commented out for now, since this is `rwl` specific checks and will be moved to `rwl` in the revised PR

Reply:
This actually helps a lot because I was trying to make sense of `m_free_log_entries` logic for the `ssd` PR I'm working on and couldn't...  With `m_free_log_entries` out of the way I should be able to post on Monday.